From f3a7589fd336e6e015ff4fff9eecce6068917d2e Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Wed, 21 Nov 2018 14:20:36 +0800
Subject: [PATCH 5170/5242] MLK-20434: ASoC: fsl_rpmsg_i2s: add more rates in
 constraint list

commit  3879fa6b0519d815f3cc091fed20cc5ec3312fd8 from
https://source.codeaurora.org/external/imx/linux-imx.git

add more rates in constraint list, Fixes commit ee959e2c9b18
("MLK-19581-3: ASoC: fsl_rpmsg_i2s: support multipul rate and DSD format")

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
(cherry picked from commit bcc6185a392b46d2b5af315894b06e6ebaff8046)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_rpmsg_i2s.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/fsl/fsl_rpmsg_i2s.c b/sound/soc/fsl/fsl_rpmsg_i2s.c
index 901e35c..192802a 100644
--- a/sound/soc/fsl/fsl_rpmsg_i2s.c
+++ b/sound/soc/fsl/fsl_rpmsg_i2s.c
@@ -81,6 +81,7 @@ static int i2s_send_message(struct i2s_rpmsg *msg,
 }
 
 static const unsigned int fsl_rpmsg_rates[] = {
+	8000, 11025, 16000, 22050, 44100,
 	32000, 48000, 96000, 88200, 176400, 192000,
 	352800, 384000, 705600, 768000, 1411200, 2822400,
 };
-- 
1.7.9.5

